src/main/scala/stellar/sdk/inet/HorizonCursor.scala

```diff
@@ -0,0 +1,8 @@
+package stellar.sdk.inet
+
+
+sealed trait HorizonCursor {
+  val value: Long = 0
+}
+case object Now extends HorizonCursor
+case class Record(override val value: Long) extends HorizonCursor
```

Comment:
Better to have `value` part of `Record` only. `Now` doesn't need it. i.e.

```
sealed trait HorizonCursor
case object Now extends HorizonCursor
case class Record(value: Long) extends HorizonCursor
```

Reply:
Now I think of it, without `Epoch` this type can simplified to just `Option[Long]`.